src/effects/LensDistortionEffect.ts

```diff
@@ -0,0 +1,116 @@
+import { Uniform, Vector2 } from "three";
+
+import { Effect } from "./Effect.js";
+import fragmentShader from "./shaders/lens-distortion.frag";
+
+export interface LensDistortionEffectOptions {
+	distortion?: Vector2;
+	principalPoint?: Vector2;
+	focalLength?: Vector2;
+	skew?: number;
+}
+
+/**
+ * A lens distortion effect.
+ *
+ * Original shader ported from https://github.com/ycw/three-lens-distortion
+ */
+
+export class LensDistortionEffect extends Effect {
+
+	/**
+   * Constructs a new lens distortion effect.
+   *
+   * @param {Object} options - The options.
+   */
+	constructor({
+		distortion = new Vector2(0, 0),
+		principalPoint = new Vector2(0, 0),
+		focalLength = new Vector2(1, 1),
+		skew = 0
+	}: LensDistortionEffectOptions = {}) {
+
+		super("LensDistortionEffect");
+
+		this.fragmentShader = fragmentShader;
+		const uniforms = this.input.uniforms;
+		uniforms.set("distortion", new Uniform(distortion));
+		uniforms.set("principalPoint", new Uniform(principalPoint));
+		uniforms.set("focalLength", new Uniform(focalLength));
+		uniforms.set("skew", new Uniform(skew));
+
+	}
+
+	/**
+   * The radial distortion coefficients. Default is (0, 0).
+   *
+   * @type {Vector2}
```

Comment:
Remove the JSDoc type tags and specify the return type in the method declaration

Reply:
Not sure if I understood correctly but I removed all the return types from the jsdocs.

Reply:
In typescript you can specify return types which removes the need for JSDoc tags:
```ts
get distortion(): Vector2;
```
But no worries, I can just add that later :ok_hand:

Reply:
Ah, gotcha! I thought that's not needed as the type is already inferred from the return value. But thanks for adding them.